domain/src/main/java/edu/nextstep/camp/calculator/domain/validator/CalculatorInputValidator.kt

```diff
@@ -0,0 +1,24 @@
+package edu.nextstep.camp.calculator.domain
+
+class CalculatorInputValidator {
+    companion object {
+        fun isValidOperationMark(text: String): Boolean {
+            return if (CalculatorInputUtil.isOperationMarkRegex(text)) {
+                true
+            } else {
+                throw IllegalArgumentException("The Mark is not operation mark use: [× - + ÷]")
+                false
+            }
+        }
+
+        fun isValueNotEmpty(text: String): Boolean {
+            return if (text != "" && text != null && text != " ") {
+                true
+            } else {
+                throw IllegalArgumentException("string is empty or null")
+                false
+            }
```

Comment:
```suggestion
            return text.isNotEmpty()
```
isNotEmpty 함수를 활용해볼 수 있을 것 같아요.